index.js

```diff
@@ -52,31 +89,26 @@ module.exports = function renderDom(rootPostOrString, options, cb) {
 			postOrString(currentPostOrString, butler, function (err, currPost) {
 				if (err) return onLoadCb(err)
 
-				augmentCurrentData(currPost, butler, { local: true }, function(err, postListData) {
-					if (err) return onLoadCb(err)
+				posts[currPost.filename] = currPost
 
-					scan(currPost, util, state, currentFilename === currPost.filename, function() {
-						var startingData = extend(BASE_DATA, options.data || {}, postListData, setCurrentData, {
-							removeDots: removeDots,
-							metadata: currPost.metadata,
-							current: currPost.filename
-						})
-
-						currentFilename = currPost.filename
+				scan(currPost, util, state, currentFilename === currPost.filename, function() {
+					var startingData = extend(BASE_DATA, options.data || {}, setCurrentData, currPost.metadata, {
```

Comment:
Very minor point, we don't need BASE_DATA here, since we have `postList` and `posts` below.

Maybe BASE_DATA should be inlined on line 52?